model-optimizer/automation/package_BOM.txt

```diff
@@ -183,6 +183,7 @@ extensions/front/mxnet/cumsum.py
 extensions/front/mxnet/cumsum_ext.py
 extensions/front/mxnet/custom.py
 extensions/front/mxnet/custom_rpn_proposal.py
+extensions/front/mxnet/deduce_parameter_shape.py
```

Comment:
why mxnet? we also have dynamicly ranked onnx and tf models. what is the nature of such restriction?

Reply:
I believed that only `mxnet` models can have dynamic rank. It would be great to see models with dynamic rank in other FWs

Reply:
There are TF models in the model zoo with dynamic input shapes as well.

Reply:
Please, take a look for a concept of a better solution https://github.com/openvinotoolkit/openvino/pull/8767/files